pandas/core/series.py

```diff
@@ -1340,8 +1341,14 @@ def idxmax(self, axis=None, skipna=True, *args, **kwargs):
         return self.index[i]
 
     # ndarray compat
-    argmin = idxmin
-    argmax = idxmax
+    argmin = deprecate('argmin', idxmin,
+        msg="argmin is deprecated. Use idxmin instead. " + \
```

Comment:
The `+ \` is not needed (the string is concatenated due to implicit line continuation because we are inside the ( .. ) of deprecate)

Reply:
Can you put single quotes around the function names argmin and idxmin? 
And can you also add the alternative for when you need the actual argmin?

(and the same comments for the argmax message below